src/main/g8/README.md

```diff
@@ -261,13 +271,4 @@ object Main extends LazyLogging {
 
 ## Who do I talk to? ##
 
-- Email: chap.s.b@gmail.com
-- Github: https://github.com/sbchapin/
-- Bitbucket: https://bitbucket.org/sbchapin/
-- Discord: sbchapin#7435
-
-## What's the future of this repo? ##
-
-- Keep this baby up to date
-- More robust examples of the existing libraries
-- CI & CD Integrations
+This project was generated from [this quickstart](https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8) by $maintainer$
```

Comment:
Having been on the receiving end of these generator results before, info that points to the person who actually ran the generator in the first place can be a lifesaver.